src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+
+        String action = null;
+        Scanner input = new Scanner(System.in);
+        Task[] allTasks = new Task[100];
+        int numTasks = 0;
+        int taskNum = 0;
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("----------------------------------------------------");
+        System.out.println(logo);
+        System.out.println("Aloha! I'm Duke");
+        System.out.println("What would you like to do today?");
+        System.out.println("Enter 'Bye' when done.");
+        System.out.println("----------------------------------------------------");
+
+        // obtain string input
+        if(!input.hasNext() || (action = input.nextLine()).isEmpty()) System.exit(0);
+
+        while (!action.equalsIgnoreCase("Bye")){
+            String actionArr[] = action.split(" ", 2);
+            switch(actionArr[0]){
+
+                case "done":
+                    taskNum = Integer.parseInt(actionArr[1]);
+
+                    taskNum = taskNum - 1;
+                    allTasks[taskNum].setDone();
+                    System.out.println("You've completed " + allTasks[taskNum].getTask() + "!");
+                    System.out.println("Your current tasks: ");
+                    for(int i=0; i < allTasks.length; i++) {
+                        if (allTasks[i] != null) {
+                            taskNum = i + 1;
+                            System.out.println(taskNum + ". " + allTasks[i].getDone() + " " + allTasks[i].getTask());
+                        }
+                    }
+                    System.out.println("----------------------------------------------------");
+                    break;
+                case "list":
+                    System.out.println("Your current tasks: ");
+                    for(int i=0; i < allTasks.length; i++) {
```

Comment:
Should have spacing between the  for loops conditions and operators line 38 and 48.
E.g. for (int i = 0; i < allTasks.length; i++) { ...